src/Resources/config/packages/shopware.yaml

```diff
@@ -8,3 +8,6 @@ shopware:
             SWAG_PAYPAL__API_CANNOT_BE_ZERO_OR_NEGATIVE: notice
             SWAG_PAYPAL__API_PAYMENT_SOURCE_INFO_CANNOT_BE_VERIFIED: notice
             SWAG_PAYPAL__API_PAYMENT_SOURCE_DECLINED_BY_PROCESSOR: notice
+    paypal:
+        webhook_proxy_url: null
```

Comment:
could you do one more layer? so
```
paypal:
    webhook:
        proxy_url: null
```
Background: we will in a future iteration, add more config parameters to the yaml files, therefore we would like to have some sort of namespacing